lib/tasks/efile.rake

```diff
@@ -23,8 +23,4 @@ namespace :efile do
     puts("End: Found #{EfileSubmission.in_state(:failed).count} failed submissions")
     exit
   end
-
```

Comment:
Forgot to remove this in the last PR when it moved from efile to state_file
https://github.com/codeforamerica/vita-min/blob/dd6d1ed7f155503f2c0dc218b58916c9b360f445/lib/tasks/state_file.rake